ceph.spec.in

```diff
@@ -527,12 +529,14 @@ Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-jwt
 Requires:       python%{python3_pkgversion}-routes
 Requires:       python%{python3_pkgversion}-werkzeug
+Requires:       python%{python3_pkgversion}-yaml
```

Comment:
strange enough, i cannot find the place where dashboard imports `yaml`.

the only two places are
```
src/pybind/mgr/k8sevents/module.py:import yaml
src/pybind/mgr/orchestrator_cli/module.py:import yaml
```